cmd/meroxa/root/connectors/connect_test.go

```diff
@@ -184,7 +183,7 @@ func TestConnectExecutionNoFlags(t *testing.T) {
 
 	err := c.Execute(ctx)
 
-	expected := "requires either a source (--from) or a destination (--to)"
+	expected := "requires pipeline name (--pipeline)"
```

Comment:
I don't think this is right. `meroxa connect ` should still require either `--from` or `--to`.

Reply:
oh, maybe this is because of moving the pipeline flag code up 20 lines in the next block